README.md

```diff
@@ -91,6 +91,11 @@ As a convenience, if you would like to install StandardFirmata you can do so by:
 interchange install StandardFirmata -a <board> -p <port>
 ```
 
+For an interactive interface that will prompt you with choices for each option, use
+```
+interchange install --interactive
+```
+
```

Comment:
Should probably include a screen shot here as well showing what that interactive version looks like,